### PR TITLE
Td refactor inventory

### PR DESCRIPTION
**Pull request by snappycodernet**
Implemented necessary functionality to finish the character/inventory screen

* added tooltips for items in inventory slots
* added ability to equip single items by clicking on them
* improved CSS styling
* fixed issue with jquery-ui not loading properly
* several other minor changes

### CODE REVIEW

File: js/ui.js [hunk 1]
```diff
@@ -115,13 +115,60 @@ function showRoom(player) {
...
+      }
+    }
+  });
+}
+
+function clearInventory() {
+  $("#equipped-weapon-text").text("");
+  $("#equipped-armor-text").text("");
+
+  for(i = 0; i < player.items.length; i++) {
```
Comment (atrotter0) [inline js/ui.js]: You could probably get rid of this loop if you add a class to all slots. Then you can just reference the class with jquery and call .empty();

File: js/item.js [hunk 2]
```diff
@@ -1,7 +1,6 @@
 // Items
 console.log("item.js loaded!");
 
-var itemMap = {}; // get rid of this?
```
Comment (atrotter0) [inline js/item.js]: we use ItemMap everywhere in develop currently. Can this get added back?

File: js/ui.js [hunk 3]
```diff
@@ -115,13 +115,60 @@ function showRoom(player) {
...
+
 function displayEquippedItems(player) {
+  clearInventory();
   $("#equipped-weapon-text").text(player.equippedWeapon.name);
   $("#equipped-armor-text").text(player.equippedArmor.name);
 
   for(i = 0; i < player.items.length; i++) {
+    var itemName = player.items[i].name;
+    var itemLevel = player.items[i].level;
+    var hp = player.items[i].healthBonus;
```
Comment (atrotter0) [inline js/ui.js]: `healthBonus` has been changed back to `addHp` on develop.

File: js/ui.js [hunk 4]
```diff
@@ -115,13 +115,60 @@ function showRoom(player) {
...
+    var ap = player.items[i].attackBonus;
+    var sp = player.items[i].spellBonus;
+    var mp = player.items[i].manaBonus;
```
Comment (atrotter0) [inline js/ui.js]: `manaBonus` has been changed back to `addMp` on develop.

https://github.com/atrotter0/rpg-group-project/blob/develop/js/item.js#L7

**Review by atrotter0 — CHANGES_REQUESTED** [review]
Just a few things you need to update before we commit this one. Please update and resubmit :)

File: js/ui.js (hunk 1 above)
Comment (snappycodernet) [reply js/ui.js]: the slots all have a grid-item class. the loop is needed to initialize the tooltip attributes for each inventory item and appending them to each item using jquery. i don't see any way around that. do you have an example?

Comment (atrotter0) [reply js/ui.js]: This loop on line 140 is merely emptying out every div with `#slot` id attribute right? I think you can do the same thing by just assigning a class to those divs (or referencing an existing class) and calling `.empty()` with jquery. So you might not need this loop!

`$(".some-class").empty();`

**Review by atrotter0 — APPROVED** [review]

**Pull request merged**